packages/web/src/components/gcds-select/gcds-select.tsx

```diff
@@ -90,14 +92,38 @@ export class GcdsSelect {
    */
   @Prop({ reflect: true, mutable: false }) defaultValue?: string;
 
+
+  /**
+   * If true, the select will be focused on component render
+   */
+  @Prop({ reflect: true }) autofocus: boolean;
+
+  /**
+   * The ID of the form that the select field belongs to.
+   */
+  @Prop({ reflect: true }) form?: string;
+
+  /**
+   * String to have autocomplete enabled.
+   */
+  @Prop() autocomplete?: string;
+
   /**
    * Value for a select element.
    */
   @Prop({ mutable: true }) value?: string;
 
   @Watch('value')
   watchValue(val) {
-    this.internals.setFormValue(val ? val : null);
+    if (this.checkIfValidValue(val)) {
+      this.internals.setFormValue(val ? val : null);
```

Comment:
The check for this would have been moved up a level, so at this point we just need to set it
```suggestion
      this.internals.setFormValue(val);
```